### PR TITLE
:bug: Propagate dep labels.

### PR DESCRIPTION
**Pull request by jortel**
Propagate dep provider and labels.

### CODE REVIEW

**Review by shawn-hurley — APPROVED** [review]
LGTM

Do we have to have an associated test update?

File: builder/deps.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ func (b *Deps) Write(writer io.Writer) (err error) {
 					Name:     d.Name,
 					Version:  d.Version,
 					SHA:      d.ResolvedIdentifier,
+					Labels:   d.Labels,
```
Comment (ibolton336) [inline builder/deps.go:48]: Also, seems  the "provider" field on the dependency is always blank for any analysis.

**Review by ibolton336 — CHANGES_REQUESTED** [review]

File: builder/deps.go (hunk 1 above)
Comment (jortel) [reply builder/deps.go:48]: Fixed.

**Comment by jortel** [comment]
> LGTM
> 
> Do we have to have an associated test update?

This would be an integration test which don't live in this project.

**Comment by shawn-hurley** [comment]
I agree, but shouldn't we have a test PR to the test repo and to test them together?

**Comment by jortel** [comment]
> I agree, but shouldn't we have a test PR to the test repo and to test them together?

Ah. Yes we should.

**Comment by jortel** [comment]
Test PR: https://github.com/konveyor/go-konveyor-tests/pull/63

**Pull request merged**